addons/survey/views/survey_templates_statistics.xml

```diff
@@ -113,15 +113,14 @@
             </t>
             <!-- Scoring info -->
             <t t-if="question_scoring">
-                <t t-if="question.question_type in ['simple_choice', 'multiple_choice']">
+                <t t-if="question.is_scored_question">
```

Comment:
The "correct pil" might not be visible if you have a `simple_choice` or a `multiple_choice` with `is_scored_question` set to False, which is possible (and before it was visible)